app/serializers/api/v2/serializable_provider_enrichment.rb

```diff
@@ -0,0 +1,12 @@
+module API
+  module V2
+    class SerializableProviderEnrichment < JSONAPI::Serializable::Resource
+      type 'provider_enrichment'
+
+      has_one :provider
+
+      attributes  :email, :website, :address1, :address2, :address3, :address4,
```

Comment:
What about `accrediting_body` enrichments?

Reply:
argh #644 was meant to come after this pr :face_with_head_bandage:

Reply:
https://trello.com/c/lhmNp5HF/1808-about-your-accredited-body-pull-onto-rails-backend